src/components/Header/Header.tsx

```diff
@@ -1,3 +1,21 @@
+import './Header.scss';
+import { useState } from 'react';
+import burgerMenu from './assets/burgerMenu.svg';
+import shopBag from './assets/shopBag.svg';
+
 export const Header = () => {
-  return <div>Header</div>;
+  return (
+    <header>
+      <div className="left-buttons">
+        <img src={burgerMenu} alt="burger-menu" className="mobile-only" />
+        <div className="left-button desktop-only">Shop</div>
+        <div className="left-button desktop-only">Contact</div>
+      </div>
+      <div className="right-buttons">
+        <img src={shopBag} alt="shop-bag" className="mobile-only" />
+        <div className="sign-in-button desktop-only">Sign In</div>
```

Comment:
These 'Sign In,' 'Shop,' 'Contact,' etc., should be actual links that lead users to the respective pages.